src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -57,7 +57,7 @@ export interface Props {
    * Overflow variants
    * - **inverted** changes the overflow shadow to the inverted color
    */
-  overflow?: 'inverted';
+  overflow?: 'inverted' /** AH-Strip? */;
```

Comment:
I couldn't figure out what this `'inverted'` is actually for... seems like it _could_ be something to remove but I think this is something different. @jinlee93 any thoughts?

Reply:
after chatting with @jinlee93 will remove this one since it is unused and seems to only be for inverting color

Reply:
Thanks for removing!

Reply:
Agree—this props in the `Tabs` component is actually what made me realize there's a bunch of unused props around :/